src/commands/network.mjs

```diff
@@ -87,18 +89,9 @@ export class NetworkCommand extends BaseCommand {
 
     if (config.releaseTag) {
       const rootImage = helpers.getRootImageRepository(config.releaseTag)
-      valuesArg += ` --set defaults.root.image.repository=${rootImage}`
+      valuesArg += ` --set "defaults.root.image.repository=${rootImage}"`
     }
 
-    // prepare name and account IDs for nodes
-    const realm = constants.HEDERA_NODE_ACCOUNT_ID_START.realm
-    const shard = constants.HEDERA_NODE_ACCOUNT_ID_START.shard
-    let accountId = constants.HEDERA_NODE_ACCOUNT_ID_START.num
-    let i = 0
-    config.nodeIds.forEach(nodeId => {
```

Comment:
how are we adding the nodes in the future?

Reply:
It is now handled in a centralized place (in the new `ProfileManager` class) that adds other relevant values for FST chart.